src/main/java/core/basesyntax/shop/service/Items.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.shop.service;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public enum Items {
```

Comment:
Your code had no any restrictions about fruit type and you able to easily add any new type to the your storage. So you needn't to make enum for them.